decidim-core/app/presenters/decidim/user_presenter.rb

```diff
@@ -36,6 +36,14 @@ def profile_path
       decidim.profile_path(__getobj__.nickname)
     end
 
+    def direct_messages_enabled?(context)
+      if __getobj__.respond_to?("accepts_conversation?") && !__getobj__.accepts_conversation?(context[:current_user])
+        false
+      else
+        true
+      end
```

Comment:
There's a change that this method returns nil, not sure if it is relevant but maybe something like this would be better? 
I also think that the function should return `false` by default (in case no accepts_conversation is defined, non-logged users, etc).

```suggestion
     return false unless  __getobj__.respond_to?(:accepts_conversation?)
     __getobj__.accepts_conversation?(context[:current_user])
      end
```